client/src/views/about/ChangesModal.vue

```diff
@@ -17,7 +17,7 @@
               {{ $t('app.name') }} v{{ versionChange.version }}
             </ion-text>
             <ion-text class="version-date body">
-              {{ versionChange.date }}
+              {{ timeSince(versionChange.date) }}
```

Comment:
It shouldn't use `timeSince()`, it should just display the date as is (but formatted correctly with `d` from `const {_, d} = useI18n()`).